kite-data/kite-data-crunch/pom.xml

```diff
@@ -31,8 +31,20 @@
   </description>
 
   <build>
+    <resources>
+      <resource>
+        <directory>src/test/avro</directory>
```

Comment:
I might be guilty of cargo culting here, but I pulled this pattern from kite-data-core, which has the same stanza for its avro test resources. I verified the schema is not included in the resulting jar, so it looks like the jar step is smart enough to not include items from this folder.

There may be a better way to accomplish this, but it is at least consistent with other Kite sub-projects and produces the desired outcome.